assets/css/style.scss

```
@@ -530,5 +530,20 @@ p.black-p {
 .video {
     aspect-ratio: 16 / 9;
     width: 100%;
-    max-width: 700px;
+    max-width: 700pxš
 }
+
+@media (min-width: 992px){
+   .footer .col-12.col-lg-8.px-2.px-md-0{
```

`.col-12.col-lg-8.px-2.px-md-0` 有什么用？

```suggestion
   .footer {
```